<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

 
  <arg name="robotID" default="1"/>
  <arg name="numRobots" default="3" />
  <arg name="WindowSize" default="20" />
  <arg name="DecayLambda" default="10" />
  
     
  <node name="coop_loc_track_$(arg robotID)" pkg="mh_solver_frontend" type="coop_loc_track" args="$(arg WindowSize) $(arg DecayLambda)" output="screen" respawn="true" >
    
    <param name="MY_ID" value="$(arg robotID)" type="int" />   
    
    <param name="NUM_ROBOTS" value="$(arg numRobots)" type="int" />  
    
    <param name="MAX_VERTEX_COUNT" value="150000" type="int" /> 
    
    <param name="MAX_INDIVIDUAL_STATES" value="50000" type="int" />  
    
    <param name="NUM_TARGETS" value="1" type="int" /> 
    
    <param name="NUM_TARGETS_USED" value="1" type="int" />     

    <param name="TGT_ACC_COEFF" value="100000.0" type="double" /> 
    
    <param name="TGT_INFORMATION_X" value="5.0" type="double" /> 

    <param name="TGT_INFORMATION_Y" value="5.0" type="double" />     
     
   </node>

</launch> 
